test/00.hello_world/test_serialize_driver.py

```diff
@@ -61,3 +63,35 @@ def test_device_gpu_with_compute_capability():
     print(txt)
     device2 = ft.load_device(txt)
     assert device == device2
+
+
+def test_array_no_cpu_sizeof4():
```

Comment:
As the test uses GPU, you need to mark the test with `@pytest.mark.skipif(not ft.with_cuda(), reason="requires CUDA")`, so it can be disabled when FreeTensor is not built with CUDA.